README.md

```diff
@@ -24,6 +24,7 @@ Although SQL is reasonably consistent in its implementations, there are several
 - [Exasol](https://www.exasol.com/)
 - [Hive](https://hive.apache.org/)
 - [MySQL](https://www.mysql.com/)
+- [Oracle](https://docs.oracle.com/en/database/oracle/oracle-database/21/sqlrf/index.html)
```

Comment:
Can you also add Oracle to docs/source/dialects.rst? Think it would be good to confirm scope of this dialect in there (SQL I presume and not PL/SQL? At least initially).

Reply:
🙏  I added a blurb. Let me know if that looks good or if I should add more clarity/context there.